src/backtest.py

```
@@ -37,11 +37,10 @@
 
 # Make src importable when running from repo root
 REPO_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-SRC_PATH = os.path.join(REPO_ROOT, "src")
-if SRC_PATH not in sys.path:
-    sys.path.insert(0, SRC_PATH)
+if REPO_ROOT not in sys.path:
+    sys.path.insert(0, REPO_ROOT)
 
-from models.ensemble_model import FraudDetectionEnsemble  # noqa: E402
+from src.models.ensemble_model import FraudDetectionEnsemble  # noqa: E402
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)